src/picai_prep/dcm2mha.py

```diff
@@ -525,11 +601,130 @@ def read_image_series(image_series_path: PathLike) -> sitk.Image:
 
         # convert to SimpleITK
         image: sitk.Image = sitk.GetImageFromArray(image)
-        image.SetSpacing(list(slices[0].PixelSpacing) + [slices[0].SliceThickness])
-
+        ref = slices[0]  # corresponds to the same slice as with SimpleITK
+        image.SetSpacing(list(ref.PixelSpacing) + [ref.SliceThickness])
+        image.SetOrigin(ref.ImagePositionPatient)
+        image.SetDirection(self.get_orientation_tuple_sitk(ref))
+
+        for key in ref.keys():
```

Comment:
This is partly because part of this code was obsolete: I have removed this now. The remaining code is very similar to the first part of `_collect_metadata_pydicom`, but I do not see a way to make this more concise/less repetition, without introducing unnecessary complexity.